app/services/ab_test.rb

```diff
@@ -1,7 +1,7 @@
 class AbTest
   def initialize(key, percent_on)
     @key = key
-    @percent_on = percent_on.to_s.to_i
+    @percent_on = percent_on
```

Comment:
old sets nil to zero...we good here?

Reply:
I think we are. If we have a nil config the IdentityConfig code will raise.

Reply:
It looked like this class expected `percent_on` to be a string, and then converted it to a string to convert it to an integer? The class was only referenced for recaptcha, and that isn't used (?), so I tried switching it to just expect an integer to simplify things.